src/pages/Admin/Admin.tsx

```diff
@@ -278,57 +262,6 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
               venue={venue}
               containerStyle={{ marginTop: 20 }}
             />
-            {IS_BURN && PLACEABLE_VENUE_TEMPLATES.includes(venue.template) && (
```

Comment:
I assumed that we don't need `IS_BURN` flag anymore (correct me if I'm wrong). 
If so, we can remove this part and `PLACEABLE_VENUE_TEMPLATES` becomes not used anywhere else, so we can remove it as well.

Reply:
`IS_BURN` is still used in a bunch of places, maybe it's better to remove it entirely?

Reply:
+1 if we can weed out the `IS_BURN` uses one by one, especially in this case since there are plenty of venue templates lists in `settings.ts` already